core-tests/src/test/java/org/mule/expression/ExpressionManagerTestCase.java

```diff
@@ -30,32 +24,11 @@
     public void testManager() throws Exception
     {
         MuleMessage message = new DefaultMuleMessage("test", muleContext);
-        Object o = muleContext.getExpressionManager().evaluate("function:uuid", message);
+        Object o = muleContext.getExpressionManager().evaluate("org.mule.util.UUID.getUUID()", message);
         assertNotNull(o);
-        o = muleContext.getExpressionManager().evaluate("function:now", message);
+        o = muleContext.getExpressionManager().evaluate("server.dateTime.toDate()", message);
```

Comment:
unrelated to this PR, but we should also remove the custom DateTime classes in mule-core and use java8 dates instead